clusterloader2/pkg/util/util.go

```diff
@@ -75,6 +75,15 @@ func GetStringOrDefault(dict map[string]interface{}, key string, defaultValue st
 	return value, err
 }
 
+// GetStringSliceOrDefault tries to return value from map cast to string slice type. If value doesn't exist default value is used.
```

Comment:
Could you extract changes in this file to a separate PR?

Reply:
Should I separate the rewrite of `GetMasterIp()` to `GetMasterIps() then?
As it stands, all the callers depend on `GetStringSliceOrDefault()`, i.e., https://github.com/kubernetes/perf-tests/pull/833/files#diff-1501ea581b9304b1c1a8b79122c99a6eR69

Reply:
I guess, I could keep the usage of `GetStringOrDefault` and just call `GetMasterIps()` beforehand. WOuld that be prefered?

Reply:
You can send a PR adding GetStringSLiceOrDefault method (basically all the changes in this file) separately. As it's very straighforward change it will get merged the same day. In the meantime we can continue working on this one, which is  a bit more tricky and will require more work.